src/serialization/serialize-assets.js

```diff
@@ -16,12 +16,10 @@ const serializeAssets = function (runtime, assetType, optTargetId) {
         const currAssets = currTarget.sprite[assetType];
         for (let j = 0; j < currAssets.length; j++) {
             const currAsset = currAssets[j];
-            const assetId = currAsset.assetId;
-            const storage = runtime.storage;
-            const storedAsset = storage.get(assetId);
+            const asset = currAsset.asset;
```

Comment:
While serializing, pull the assets off of the costume/sound objects. Don't try to get them out of storage.